aws-api/src/androidTest/java/com/amplifyframework/api/aws/CodeGenerationInstrumentationTest.java

```diff
@@ -45,6 +47,9 @@
     private static final String PERSON_API_NAME = "personApi";
     private static final String PROJECT_API_NAME = "projectApi";
     private static final String BLOG_API_NAME = "blogApi";
+    private static final String NOTES_WITH_AUTH_API_NAME = "notesWithAuthApi";
+
+    private static final int ONE_SECOND_OF_MILLISECONDS = 1000;
```

Comment:
Instead of defining this constant, you could consider to use `TimeUnit.SECONDS.toMilliseconds(1)` as an in-line constant - or atlernately, use a semantic label, like:
```
private static final int MAX_SUBSCRIPTION_FAILURE_TIME_MS = TimeUnit.SECONDS.toMillseconds(1);
```
which sort of achieves two purposes of documentation - how long it is, and what it purpose is

Reply:
Updated PR